Samples/Hast.Samples.Kpz/HeighmapMetaData.cs

```diff
@@ -0,0 +1,11 @@
+namespace Hast.Samples.Kpz
+{
+    public class HeighmapMetaData

```

Comment:
Inkább az alábbi, de a metódus neve ishibá.

```suggestion
    public class HeightmapMetaData
```